tile.yml

```diff
@@ -148,28 +166,12 @@ forms:
   label: Wavefront Firehose Nozzle Config
   description: Wavefront Nozzle Configuration
   properties:
-  - name: pcf_host
-    type: string
-    label: CF API Hostname
-    description: The CF API hostname such as 'https://api.YOUR-SYSTEM-DOMAIN'
-    placeholder: https://api.YOUR-SYSTEM-DOMAIN
-    optional: false
-  - name: pcf_user
-    type: string
-    label: CF UAA Username
-    description: The CF UAA username with read-only access to the firehose and cloud controller
-    optional: false
-  - name: pcf_password
-    type: secret
-    label: CF UAA Password
-    description: The CF UAA password
-    optional: false
-  - name: pcf_log_stream
-    type: string
-    label: CF API log stream url
-    description: The CF API log stream url such as 'https://log-stream.YOUR-SYSTEM-DOMAIN'
-    placeholder: https://log-stream.YOUR-SYSTEM-DOMAIN
-    optional: false
+  - name: nozzle_credentials
+    label: Wavefront Nozzle credentials
+    type: simple_credentials
+    configurable: false
+    default:
+      identity: wavefront_nozzel
```

Comment:
typo: `wavefront_nozzle`. Not sure if this is referring to something else and that also needs changed